app/services/github_authenticator.rb

```diff
@@ -61,8 +61,8 @@ def uri
 
   def post_data
     {
-      "client_id" => ENV["GITHUB_KEY"],
-      "client_secret" => ENV["GITHUB_SECRET"],
+      "client_id" => ENV["GITHUB_CLIENT_ID"],
```

Comment:
I wonder if we should register a new github app to get updated tokens? Or do we have those tokens somewhere that we can pass around the team?